src/details/ArborX_Triangle.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef ARBORX_TRIANGLE_HPP
+#define ARBORX_TRIANGLE_HPP
+
+#include <ArborX_Point.hpp>
+
+namespace ArborX
+{
+namespace Experimental
+{
+// need to add a protection that
+// the points are not on the same line.
+struct Triangle
```

Comment:
Note to self: 36 byte alignment. Not great, but don't know if playing with would make any difference.